_content/av/buddhist-mythology_sujato.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Buddhist Mythology: The Sacred and the Profane"
+authors:
+  - sujato
+subcat: course
+external_url: "https://discourse.suttacentral.net/t/bhante-sujato-buddhist-mythology-course-sydney-2019/13204"
+drive_links:
+  - "https://drive.google.com/file/d/1ynz8Zou66pCMk8Ujz0_ZWYYlFESQnEnv/view?usp=drive_link"
+  - "https://drive.google.com/file/d/1nDK2aaeWYe8-mHaI4NtC0Lz1AjpKszpN/view?usp=drive_link"
+  - "https://drive.google.com/file/d/1J9wb-j6u3pIq9RdDZyCGDZbLY4qo-AtK/view?usp=drive_link"
+course: rebirth-stories
+tags:
+  - mythology
```

Comment:
Should be just `myth`  The `mythology` tag is for non-Buddhist myths 😅 (Confusing I know, sorry)